app/services/websites/create.rb

```diff
@@ -14,7 +14,15 @@ def initialize(params, user)
     end
 
     def call
-      if params[:website][:url].to_s.include? '.'
+      already_exist = Website.find_by(url: params[:website][:url])
+      if already_exist && params[:website][:collection_id]
+        website = CollectionWebsite.where('collection_id = ?', params[:website][:collection_id])
+                                   .where('website_id = ?', already_exist.id)
+                                   .first
+        if website.nil?
+          CollectionWebsite.create!(collection_id: params[:website][:collection_id], website_id: already_exist.id)
+        end
```

Comment:
This can be replaced with:

```ruby
website =
  CollectionWebsite
    .where(collection: params[:website][:collection_id], website: already_exist)
    .first_or_create  
```